lib/Ohmbrewer_Sprouts.cpp

```diff
@@ -101,6 +103,11 @@ int Ohmbrewer::Sprouts::addSprout(String argsStr) {
         return errorCode;
     }
 
+    if(_sprouts->size() == 1) {
+        // We should only have to start the update timer if this is the first Sprout
```

Comment:
Correct. If size is >1, it's already running periodically (check the periodic update method - I restart the timer at the end) and if size == 0, I stop the timer. The alternatives would have been to start the Timer once in the constructor, or sometime before I add it to the Sprouts object.